boogiestats/templates/boogie_ui/scorecard.html

```diff
@@ -0,0 +1,172 @@
+{% load mathfilters %}
+<div class="text-center">
+    <canvas id="statcard" width="400" height="600"></canvas>
+</div>
+<script>
+    function getLines(ctx, text, maxWidth) {
+        let words = text.split(" ");
+        let lines = [];
+        let currentLine = words[0];
+
+        for (let i = 1; i < words.length; i++) {
+            let word = words[i];
+            let width = ctx.measureText(currentLine + " " + word).width;
+            if (width < maxWidth) {
+                currentLine += " " + word;
+            } else {
+                lines.push(currentLine);
+                currentLine = word;
+            }
+        }
+        lines.push(currentLine);
+        return lines;
+    }
+
+    const canvas = document.getElementById('statcard')
+    const context = canvas.getContext('2d')
+    const nodec = '{{ score.comment }}'.includes('No Dec/WO')
+    let diffColor
+    switch ('{{ score.song.chart_info.diff }}') {
+        case 'Beginner':
+            diffColor = '#a355b8'
+            break
+        case 'Easy':
+            diffColor = '#1ec51d'
+            break
+        case 'Medium':
+            diffColor = '#d6db41'
+            break
+        case 'Hard':
+            diffColor = '#ba3049'
+            break
+        case 'Challenge':
+            diffColor = '#2691c5'
+            break
+        default:
+            diffColor = '#ffffff'
+            break
+    }
+
+    barWidth = 306
+
+    const sum = parseInt('{{ score.total_steps }}')
+    const p = parseInt('{{ score.fantastics_plus }}') / sum * barWidth;
+    const f = parseInt('{{ score.fantastics }}') / sum * barWidth;
+    const e = parseInt('{{ score.excelents }}') / sum * barWidth;
```

Comment:
```suggestion
    const e = parseInt('{{ score.excellents }}') / sum * barWidth;
```

![image](https://github.com/florczakraf/boogie-stats/assets/9034451/9adf8170-af12-4e76-b87b-cf2c241135c5)